src/pages/workspace/WorkspaceSidebar.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import {View, ScrollView} from 'react-native';
+import Navigation from '../../libs/Navigation/Navigation';
+import ROUTES from '../../ROUTES';
+import styles from '../../styles/styles';
+import Text from '../../components/Text';
+import {
+    Wallet,
+    Users,
+    Pencil,
+} from '../../components/Icon/Expensicons';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import MenuItem from '../../components/MenuItem';
+import WorkspaceDefaultAvatar from '../../../assets/images/workspace-default-avatar.svg';
+import themedefault from '../../styles/themes/default';
+import Icon from '../../components/Icon';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import withWindowDimensions, {windowDimensionsPropTypes} from '../../components/withWindowDimensions';
+import compose from '../../libs/compose';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+    ...windowDimensionsPropTypes,
+};
+
+const WorkspaceSidebar = ({translate, isSmallScreenWidth}) => {
+    const menuItems = [
+        {
+            translationKey: 'workspace.common.card',
+            icon: Wallet,
+            action: () => {
+                Navigation.navigate(ROUTES.WORKSPACE_CARD);
+            },
+            isActive: Navigation.isActive(ROUTES.WORKSPACE_CARD),
+        },
+        {
+            translationKey: 'common.people',
+            icon: Users,
+            action: () => {},
+            isActive: false,
+        },
+    ];
+
+    return (
+        <ScreenWrapper style={[!isSmallScreenWidth ? styles.borderRight : {}]}>
+            <ScrollView
+                bounces={false}
+                contentContainerStyle={[
+                    styles.flexGrow1,
+                    styles.flexColumn,
+                    styles.justifyContentBetween,
+                ]}
+            >
+                <View style={[styles.flex1]}>
+                    {isSmallScreenWidth
+                        && (
+                            <HeaderWithCloseButton
+                                title={translate('workspace.common.workspace')}
+                                onBackButtonPress={() => Navigation.dismissModal(false, false)}
+                                onCloseButtonPress={() => Navigation.dismissModal(false, false)}
+                                shouldShowBackButton
```

Comment:
It seems incorrect to have both a back button and a close button as they both do the same thing. Let's get rid of the back button in this case.